UPGRADING.md

```diff
@@ -0,0 +1,57 @@
+## Upgrading Plugins to OpenSearch/OpenSearch Dashboards
+
+For the plugins to work with OpenSearch here are all the steps you can take and get them building, passing all the tests.
```

Comment:
Nit on the English, switch 

"These are all the steps to upgrade plugins to work with OpenSearch and OpenSearch Dashboards, including building, and passing all tests."

Reply:
Got it, makes sense.